dev/protos/update.sh

```diff
@@ -38,8 +38,8 @@ PBTS="$(npm bin)/pbts"
 pushd "$WORK_DIR"
 
 # Clone necessary git repos.
-git clone https://github.com/googleapis/googleapis.git
-git clone https://github.com/google/protobuf.git
+git clone --depth 1 https://github.com/googleapis/googleapis.git
+git clone --depth 1 https://github.com/google/protobuf.git
```

Comment:
Do these changes make this script fast? That would so exciting :)

Reply:
Yep, way faster.